### PR TITLE
feat: remove some selectors used for Xcode 11 and lower

### PR DESCRIPTION
**Pull request by KazuCocoa**
Removed selector methods that are no longer used in Xcode 12+ env.

### CODE REVIEW

File: WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h [hunk 1]
```diff
@@ -26,15 +26,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)fb_deactivateWithDuration:(NSTimeInterval)duration error:(NSError **)error;
 
-/**
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h:29]: what is the reason to remove this helper? It has only been added recently

File: WebDriverAgentLib/Utilities/FBXCodeCompatibility.h [hunk 2]
```diff
@@ -34,8 +34,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface XCElementSnapshot (FBCompatibility)
 
-- (nullable XCElementSnapshot *)fb_rootElement;
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Utilities/FBXCodeCompatibility.h:37]: Nothing referred to this attribute

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
Thanks for taking care of this @KazuCocoa

**Comment by KazuCocoa** [comment]
Will create a version tonight with this. (maybe 4.1.0)
Then, will work on updating headers based on Xcode 12's package first

**Comment by KazuCocoa** [comment]
btw, with this change, current appium/WDA's selectors are mainly actions to build method name by string, and find it with selector to reduce code.

**Pull request merged**